addons/note/models/note.py

```diff
@@ -82,6 +83,10 @@ def name_create(self, name):
     @api.model
     def read_group(self, domain, fields, groupby, offset=0, limit=None, orderby=False, lazy=True):
         if groupby and groupby[0] == "stage_id":
+            if len(groupby) > 1 and not lazy:
```

Comment:
Btw, isn't better to pass execution to super in the unsupported case?

```suggestion
        if groupby and len(groupby) == 1 and groupby[0] == "stage_id" and not lazy:
```